### PR TITLE
CRAB-42491: Updating readme to clarify environment setup

### PR DESCRIPTION
**Pull request by cherrera2001**
Merge Criteria

Primary reviewer: @andre-rigon 
Knowledge base: N/A
Compatibility breaks: N/A
Security considerations pursuant to our SAMM model: N/A
Exploratory functional testing: Ran through instructions
Regression testing Tests were updated: N/A
Product Owner acceptance: @cherrera2001
Presentation slide: N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,42 +8,31 @@ and facilitate access to data in Seeq.
...
 Before doing anything else, we recommend that you build the connector template and ensure that it is fully working with
-your private system.
+your system.
 
 From the root directory, execute the `build` command. This command will download dependencies from the web, so make
-sure you have a good internet connection. If it fails for some (non-obvious) reason, email the output (including the
-error message) to [support@seeq.com](mailto:support@seeq.com).
-
-Make sure your private Seeq Server is running on this machine.
+sure you have a good internet connection. If you have any non-obvious issues building this project. Please post your
```
Comment (andre-rigon) [inline README.md:30]: Typo: I think `project. Please` was meant to be `project, please`

File: README.md [hunk 2]
```diff
@@ -66,8 +55,11 @@ Take the following steps to verify your debugging setup:
 
 1. Open the `src/main/java/com/seeq/link/sdk/debugging/Main.java` file in the `seeq-link-sdk-debugging-agent` project.
 1. Modify the URL on the line `String seeqUrl = "https://yourserver.seeq.host";` to match your Seeq server
+1. Retrieve the agent_api_key from your Seeq Server by loggin in as a Seeq Administrator and navigating to the API
```
Comment (andre-rigon) [inline README.md:58]: Typo: `loggin` -> `logging`

File: README.md [hunk 3]
```diff
@@ -112,18 +104,21 @@ to the debug console and to the `java/seeq-link-sdk-debugging-agent/build/log/jv
...
-file into the `plugins/connectors` folder within Seeq's `data` folder. (The data folder is usually
-`C:\ProgramData\Seeq\data` on Windows and `~/.seeq/data` on Ubuntu and OSX.) You should end up with one new folder in
-`plugins/connectors`. For example, if you kept the default name for the connector, you would have a
-`plugins/connectors/mycompany-seeq-link-connector-myconnector` folder with a jar file inside.
+1. Shut down the Seeq Remote Agent - execute `seeq stop` in the Seeq CLI
+1. Verify that you have enabled support for plugins
+   1. execute `seeq config get Features/Plugins/Enabled` in the Seeq CLI
+   1. Ensure the value is `True`
+   1. If it is not then execute `seeq config set Features/Plugins/Enabled True`
+1. Copy the generate zip file to the `plugins/connectors` folder within Seeq's `data` folder (The data folder
```
Comment (andre-rigon) [inline README.md:115]: Typo: `generate` -> `generated`

**Review by andre-rigon — APPROVED** [review]
Looks great to me, I left only some minor comments about typos.

Thanks for updating this, Chris!

File: README.md [hunk 4]
```diff
@@ -112,18 +104,21 @@ to the debug console and to the `java/seeq-link-sdk-debugging-agent/build/log/jv
...
 created in the `build/distributions` folder of your connector.
 
-Copy this zip file to the Seeq Server you wish to deploy it to. Shut down the server and extract the contents of the zip
-file into the `plugins/connectors` folder within Seeq's `data` folder. (The data folder is usually
-`C:\ProgramData\Seeq\data` on Windows and `~/.seeq/data` on Ubuntu and OSX.) You should end up with one new folder in
-`plugins/connectors`. For example, if you kept the default name for the connector, you would have a
-`plugins/connectors/mycompany-seeq-link-connector-myconnector` folder with a jar file inside.
+1. Shut down the Seeq Remote Agent - execute `seeq stop` in the Seeq CLI
+1. Verify that you have enabled support for plugins
+   1. execute `seeq config get Features/Plugins/Enabled` in the Seeq CLI
```
Comment (mar1u50) [inline README.md:112]: I don't think this is needed. IMHO Features/Plugins/Enabled is only used for plugins (like paralel coordinates for examples). The remote agent is looking into plugins/connectors anyway.

Comment (andre-rigon) [reply README.md:112]: I think you're right Marius, I didn't know this!

**Pull request merged**